clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriter.java

```diff
@@ -352,6 +394,23 @@ public boolean validateBatchIngestion() {
     return true;
   }
 
+  /**
+   * If there are files already ingested in DB and we get here, then it means that the ingestion started but faced issues
+   * before completion or the SN crashed before the status of EOP was synced to OffsetRecord. In both these cases,
```

Comment:
Would recommend `s/SN/process/` since this could be Da Vinci. Also, we haven't been super heavy on the use of "SN" so far, usually favoring "server", so I would prefer we don't introduce new usages of "SN".

Reply:
Yes, cleaned this up.